src/Check/ScrutinizerCheck.php

```diff
@@ -6,8 +6,10 @@
 use SilverStripe\ModuleRatings\Check;
 
 class ScrutinizerCheck extends Check
-{
+{   
+    // phpcs:disable
     const THRESHOLD = 6.5;
+    // phpcs:enable
```

Comment:
```suggestion
{
    const THRESHOLD = 6.5;
```
The phpcs issue was resolved in another PR which has since been merged.